src/modules/common/ImageDisplay.tsx

```diff
@@ -0,0 +1,71 @@
+import { TransformComponent, TransformWrapper } from 'react-zoom-pan-pinch';
+
+import FindReplaceIcon from '@mui/icons-material/FindReplace';
+import { Box, Button, Stack } from '@mui/material';
+
+import { useNameFrameTranslation } from '@/config/i18n';
+import { hooks, mutations } from '@/config/queryClient';
+import { NAME_THE_FRAME } from '@/langs/constants';
+
+const ImageDisplay = ({
+  appSettingId,
+}: {
+  appSettingId: string;
+}): JSX.Element => {
+  const { t } = useNameFrameTranslation();
+
+  const { data: dataFile } = hooks.useAppSettingFile({ appSettingId });
+  const { mutate: deleteAppSettings } = mutations.useDeleteAppSetting();
+
+  const deleteImage = (): void => {
+    deleteAppSettings({ id: appSettingId });
+  };
+
+  return (
+    <Stack spacing={1}>
+      <Box>
+        <Button
+          variant="outlined"
+          startIcon={<FindReplaceIcon />}
+          onClick={deleteImage}
+        >
+          {t(NAME_THE_FRAME.REPLACE_IMAGE)}
```

Comment:
Should this really go in common? It looks like a component only used for builder view.